README.md

```diff
@@ -2,7 +2,7 @@
 This is effectively part of the existing HoloToolkit, but this is the repository that will contain all Unity specific components.
 The HoloToolkit is a collection of scripts and components intended to accelerate development of holographic applications targeting Windows Holographic.
 
-**Required Unity Editor Version: 5.6.2f1**
+**Required Unity Editor Version: 2017.1.0f3**
```

Comment:
Just thought of this to address some feedback I got from folks.
Could you please add a Unity link here so people know which installer to use exactly? Thanks!